src/index.js

```diff
@@ -68,6 +75,15 @@ const Restypie = module.exports = {
 
   arrayToList(arr) { return (arr || []).join(this.LIST_SEPARATOR); },
 
+  getItemInPath(path, index, separator) {
+    if (!path) {
+      return null;
+    }
+    index = index || 0;
+    separator = separator || this.POPULATE_PATH_SEPARATOR;
```

Comment:
`.` is also used for deep filtering too. So you probably want this constants to be something like `DEEP_PATH_SEPARATOR`